wandb/apis/public/api.py

```diff
@@ -301,6 +320,7 @@ def create_report(
         width: Optional[str] = "readable",
         blocks: Optional["wandb.apis.reports.util.Block"] = None,
     ) -> "wandb.apis.reports.Report":
+        """Create a new report."""
```

Comment:
why only partial if already touching this?

Reply:
Mainly just an oversight -- thanks for catching this (and the others)!